### PR TITLE
Update OWNERS file

### PR DESCRIPTION
**Pull request by carlisia**
~This repo doesn't rely on bots to manage permissions, so this because
obsolete.~

Clean up perm related files. Since the linter won't let us delete the OWNERS file.

Signed-off-by: Carlisia <carlisia@vmware.com>

### CODE REVIEW

**Comment by cpanato** [comment]
Need to bump the chart version

**Comment by carlisia** [comment]
Hey @cpanato when you have a chance, might as well add yourself here: https://github.com/vmware-tanzu/helm-charts/pull/111/files#diff-e913a658a240078f9cfb97d5d730fba0

I didn't add it bc I didn't find an email addy for you.

File: charts/velero/OWNERS [hunk 1]
```diff
@@ -1,10 +1,12 @@
 approvers:
-- hectorj2f
+- skriss
 - nrb
 - carlisia
 - ashish-amarnath
+- cpanato
 reviewers:
-- hectorj2f
+- kriss
```
Comment (skriss) [inline charts/velero/OWNERS:8]: `skriss`

File: charts/velero/OWNERS [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 - carlisia
 - ashish-amarnath
+- cpanato
 reviewers:
-- hectorj2f
+- skriss
 - nrb
 - carlisia
 - ashish-amarnath
+- cpanato
```
Comment (cpanato) [inline charts/velero/OWNERS:12]: @carlisia my email is `ctadeu[at]gmail.com`

can we make this list in alphabetical order or there is a reason to be like this?

Comment (carlisia) [reply charts/velero/OWNERS:12]: Good point! And I'll add you.

**Comment by carlisia** [comment]
@nrb / @cpanato why are we getting this message: https://github.com/vmware-tanzu/helm-charts/pull/111/checks?check_run_id=717514102#step:6:202

**Comment by carlisia** [comment]
I re-ran a couple times and it's fine now.

**Comment by cpanato** [comment]
@carlisia I will do some debugs, but maybe some issues with Kind clusters, i will try to reproduce that locally

**Pull request merged**